build_projects/shared-build-targets-utils/Publishing/AzurePublisher.cs

```diff
@@ -22,7 +22,7 @@ public class AzurePublisher
 
         public AzurePublisher()
         {
-            _connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING").Trim('"');
+            _connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING")?.Trim('"');
```

Comment:
Does this work later?  Or do you just get an less specific error later in the execution?

Maybe this should use EnvVars.EnsureVariable, so we get a good error message that CONNECTION_STRING is a required env var.